app/index.html

```diff
@@ -4,28 +4,51 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
     <title>Boats Animator</title>
-    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/font-awesome.min.css">
+    <link rel="stylesheet" href="css/fonts.css">
+    <link rel="stylesheet" href="css/animator.css">
 </head>
 
-<body class="index" onload="canDisplayNews()">
-    <header class="app-info">
-        <h1>Boats Animator</h1>
-        <footer>Copyright (C) Charlie Lee 2015 <span style="float: right; text-align: right; padding: 0 4%;"><a href="#" onclick="gui.Shell.openExternal('http://charlielee.uk')">charlielee.uk</a></span></footer>
+<body class="animator" onload="canDisplayNews()">
+    <!--Sidebar content-->
+    <div id="sidebar">
+        <div id="sidebar-content">
 
-        <h2>Recent News</h2>
-        <div class="container-news"></div>
-    </header>
-
-    <div class="indexContent">
-        <h2>Project</h2>
-        <button onclick="openAnimator()">New Project</button>
-        <button onclick="alert('This feature is coming soon!')">Load Project</button>
-        <button onclick="dev()">Dev tools</button>
-        <button onclick="reload()">Refresh</button>
-        <h2>Recent Projects</h2>
-        <div class="list" id="recentProjectsList">This feature is coming soon!</div>
+            <!--Capture Options-->
+            <div class="sidebar-item">
+                <div class="sidebar-header">
+                    <h2><i class="fa fa-newspaper-o"></i> Recent News</h2>
+                </div>
+                <ul>
+                    <li><div class="container-news"></div></li>
+                </ul>
+            </div>
+        </div>
+    </div>
+    <div id="main-event">
+        <div class="content">
+            <h2>Projects</h2>
+            <ul id="projects-container">
+                <li><i class="fa fa-folder-open-o"></i> <a href="#">Recent Project 1</a></li>
+                <li><i class="fa fa-folder-open-o"></i> <a href="#">Recent Project 2</a></li>
+                <li><i class="fa fa-folder-open-o"></i> <a href="#">Recent Project 3</a></li>
```

Comment:
Be aware this HTML will be dynamically injected instead of hard-coded like this. You'll just want to comment out (instead of `display: none`) the projects stuff right before merging.